nixos/modules/services/misc/gollum.nix

```diff
@@ -165,7 +166,12 @@ in
             ${cfg.stateDir}
         '';
       };
-    };
+    } // (lib.optionalAttrs cfg.init {
```

Comment:
This should use lib.mkIf

Reply:
I.e. 

~~~
    systemd.services.gollum = {
      description = "Gollum wiki";
      after = [ "network.target" ];
      wantedBy = [ "multi-user.target" ];
      path = [ pkgs.git ];
      preStart = lib.mkIf cfg.init
      ''
        # This is safe to be run on an existing repo except if it is bare
        git init ${cfg.stateDir}
      '';

~~~

instead?